### PR TITLE
finishing the dark theme implementation and toggle

### PR DESCRIPTION
**Pull request by sumitarora**
should fix: https://github.com/rangle/augury/issues/378

### CODE REVIEW

**Comment by igor-ka** [comment]
@sumitarora @vanessayuenn @danielschifano this looks good to me, but some of the code here is not my strong suite, i.e. the CSS part. I can give it a 👍 let's have a second pair of eye on this just in case.

@clbond can you have a look and give a 👍 if it looks good? I'll complete the review then.

**Comment by clbond** [comment]
@igor-ka Absolutely, will do

File: src/frontend/components/header/header.html [hunk 1]
```diff
@@ -1,42 +1,76 @@
...
+        <label for="light">Light Mode</label>
+      </li>
+      <li class="p3">
+        <input type="radio"
+          #darkTheme
+          [checked]="theme==='dark'"
+          (change)="themeChange('dark', darkTheme.checked)"
+          name="theme"
+          value="dark">
+        <label for="dark">Dark Mode</label>
```
Comment (clbond) [inline src/frontend/components/header/header.html]: Is it not possible to hook this up to the Chrome DevTools setting instead of duplicating it with our own setting?

File: src/frontend/frontend.ts [hunk 2]
```diff
@@ -151,6 +158,11 @@ class App {
       this.searchDisabled = false;
     }
   }
+
+  themeChanged(newTheme) {
+    this.theme = newTheme;
+    // TODO: do stuff here to persist the theme selection
```
Comment (clbond) [inline src/frontend/frontend.ts]: Does this mean I will have to re-select dark theme each time I reopen Chrome / Augury?

File: src/frontend/components/header/header.html [hunk 3]
```diff
@@ -1,42 +1,76 @@
 <div class="flex flex-column flex-stretch col-12">
-  <header class="bg-darken-1 border-bottom py3 px3">
-    <h4 class="regular flex flex-center primary-color col col-4 m0">
+  <header class="flex flex-justify bg-panel border-bottom py3 px3 dark-theme">
```
Comment (clbond) [inline src/frontend/components/header/header.html]: I can't see where this .dark-theme style is defined - is it defined? It looks like the right thing to use is "dark" or "light", not "dark-theme" - or am I misunderstanding?

**Comment by clbond** [comment]
Comments are above. Not sure if any of these issues need to be addressed before merging.

File: src/frontend/components/header/header.html (hunk 1 above)
Comment (sumitarora) [inline src/frontend/components/header/header.html]: chrome's dev api doesn't expose the flag for that

File: src/frontend/frontend.ts (hunk 2 above)
Comment (sumitarora) [inline src/frontend/frontend.ts]: yes @clbond look into storing the values in local storage and reload it

File: src/frontend/components/header/header.html (hunk 1 above)
Comment (clbond) [inline src/frontend/components/header/header.html]: I think I found a way to extract this setting. I will take a shot after this PR is merged

File: src/frontend/frontend.ts (hunk 2 above)
Comment (igor-ka) [inline src/frontend/frontend.ts]: @sumitarora let's do this in a separate PR. @clbond can you open an issue for the storing the value of this setting?

**Comment by igor-ka** [comment]
@clbond let's create separate issues based in the discussion in the comments above. Otherwise, looks like @sumitarora addressed the code issue, so :shipit:

**Comment by sumitarora** [comment]
done created new issue: https://github.com/rangle/augury/issues/423
